.circleci/config.yml

```diff
@@ -134,7 +134,6 @@ jobs:
     <<: *defaults
     parameters:
       version:
-        default: '14.19.0'
```

Comment:
Unnecessary as we'll always test against a specific version and it also means updating the default version in two places